src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/CartController.java

```diff
@@ -245,7 +239,6 @@ public AddToCartResponse addSamplesToCart(@RequestBody AddToCartRequest addToCar
 	 * @return a map stating success
 	 */
 	@RequestMapping(value = "/project/{projectId}", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
-	@ResponseBody
```

Comment:
This method still returns something does it not?  Should the `@ResponseBody` still be here?

Reply:
This method actually does not need to return anything like the others.  Changed the return type to void and removed return statement in 420756e43e614bb700c02b4fdb79a90d343ec139